src/algebra/triv_sq_zero_ext.lean

```diff
@@ -12,7 +12,13 @@ import linear_algebra.prod
 
 Given a module `M` over a ring `R`, the trivial square-zero extension of `M` over `R` is defined
 to be the `R`-algebra `R ⊕ M` with multiplication given by
-`(r₁ + m₁) * (r₂ + m₂) = r₁ r₂ + r₁ m₂ + r₂ m₁`.
+`(r₁ + m₁) * (r₂ + m₂) = r₁ r₂ + r₁ m₂ + m₁ r₂`.
+
+Note that expressing this requires bimodules; we write these as
```

Comment:
I think I would have understood this comment more quickly if it was even more explicit, perhaps by including something like the following:
```lean
example (R' M : Type*) [comm_semiring R'] [add_comm_monoid M] [module R' M] [module R'ᵐᵒᵖ M]
  [is_central_scalar R' M] : smul_comm_class R' R'ᵐᵒᵖ M :=
by apply_instance
```